src/Pate/Verification/StrongestPosts.hs

```diff
@@ -157,11 +157,57 @@ pairGraphComputeFixpoint gr =
   case chooseWorkItem gr of
     Nothing -> return gr
     Just (gr', nd, preSpec) -> do
-      gr'' <- PS.viewSpec preSpec $ \vars asm d -> do
-        withAssumptionSet asm $
-          visitNode nd vars d gr'
+      gr'' <- PS.viewSpec preSpec $ \scope d -> do
+        withAssumptionSet (PS.scopeAsm scope) $
+          visitNode scope nd d gr'
       pairGraphComputeFixpoint gr''
 
+
+
+absValueToAsm ::
+  forall sym arch v bin tp.
+  PS.SimVars sym arch v bin ->
+  PSR.MacawRegEntry sym tp ->
+  MAS.AbsValue (MM.ArchAddrWidth arch) tp ->
+  EquivM sym arch (PS.AssumptionSet sym v)
+absValueToAsm vars regEntry val = withSym $ \sym -> case val of
+  -- FIXME: check the MemAddr here to make sure we only use
+  -- stack offsets from this frame
```

Comment:
Which `MemAddr` is this referring to? Is that the `_` in the pattern match on the `StackOffsetAbsVal`?

Reply:
Yes, I'll bind it to something just so it's not mysterious.